model/src/contract.rs

```diff
@@ -474,6 +522,7 @@ where
 
     fn deploy(&mut self, input: Self::DeployArgs) {
         self.deploy_with(&input.encode()[..])
+            .expect("`deploy` failed to execute properly")
```

Comment:
The `TestableContract` could also return the error codes so we could, well, test them.

Reply:
Yeah but this is the wrong layer in my opinion because normally you do not test the `ink_model` layer and if it behaves properly upon incorrect input but instead you want to test your smart contract with valid input. Also this is guarded by `ink_lang`.